model/postgres/post.sql

```diff
@@ -0,0 +1,24 @@
+ALTER SCHEMA old RENAME TO bultdatabasen;
+
+ALTER TABLE resource ALTER COLUMN counters SET DEFAULT json_build_object();
+
+create extension ltree;
+
+CREATE TABLE bultdatabasen.tree (
+    resource_id character varying(36) NOT NULL,
+    path ltree NOT NULL
+);
+
+CREATE INDEX path_gist_idx ON tree USING GIST (path);
+CREATE INDEX path_idx ON tree USING BTREE (path);
+
+CREATE OR REPLACE FUNCTION
+  populate_path(ID varchar)
+RETURNS
+  void
+AS $$
+DECLARE
+
+BEGIN
+  RAISE NOTICE 'A simple text';
+END $$ LANGUAGE plpgsql;
```

Comment:
What purpose does this fill?

Reply:
Nothing at the moment. Will make this a procedure that builds up the new ltree paths.

Reply:
Take a look now! :wink: